### DISCUSSION TITLE
No CDN's please

### DISCUSSION THREAD

**Discussion by MACscr** [filamentphp/filament#3446] [Ideas]
Would love for all core assets to be local. So no CDN's for css or javascript.

**Comment by danharrin** [comment]
The only CDNs are for third party component translations ATM

**Reply by MACscr** [reply]
I understand that, but:

1) I don't need any translations as I only support the English language
2) Their purpose really has no bearing on my (and many others) need to not use external assets, especially js.

**Reply by danharrin** [reply]
The i18n CDNs have been removed in #3475.

**Comment by OvalMedia** [comment]
By default filament is loading google fonts as well. They could (should?) be made local.

**Reply by danharrin** [reply]
You can change this URL in the config to something local

**Reply by OvalMedia** [reply]
I was suggesting to make them local by default.
